internal/corazawaf/transaction_test.go

```diff
@@ -1360,6 +1378,18 @@ func makeTransaction(t testing.TB) *Transaction {
 	return tx
 }
 
+func makeTransactionTimestamped(t testing.TB) *Transaction {
```

Comment:
I wonder by testing.TB

Reply:
The same interface as in `makeTransaction` and `makeTransactionMultipart` to ensure uniformity.